src/dnddiceroller/DiceTower.java

```diff
@@ -16,13 +16,45 @@
  */
 public class DiceTower {
   final int PANEL_COUNT = 3;
-  List<Die> dice;
-
+  private List<NumberedDie> numberedDice;
+  private List<Load> loadDice;
+  private int trayValue;
+  private List<Fudge> fudgeDice;
+  
+  
+  //constructor for array list
   public DiceTower() {
-	this.dice = new ArrayList();
```

Comment:
all of your dice now share `Die` as an ancestor so one since ArrayList can hold all kinds of dice